app/src/main/res/values/strings.xml

```diff
@@ -194,6 +194,7 @@
     <!-- Software Keyboard -->
     <string name="input_hint">Input Text</string>
     <!-- Misc -->
+    <string name="copied_to_clipboard">Copied to clipboard</string>
```

Comment:
Don't make a new Misc section for this, but move this at the top of the file, in the Common section.

Reply:
> Don't make a new Misc section for this, but move this at the top of the file, in the Common section.

It's not new btw, it was just an empty section

Reply:
Right, in that case delete it, and move to Common.

Reply:
Done